ci/kokoro/windows/builds/quickstart-cmake.ps1

```diff
@@ -80,8 +78,7 @@ ForEach($feature in $features) {
         "-DCMAKE_TOOLCHAIN_FILE=`"${vcpkg_root}/scripts/buildsystems/vcpkg.cmake`""
         "-DCMAKE_BUILD_TYPE=${env:CONFIG}",
         "-DVCPKG_TARGET_TRIPLET=${env:VCPKG_TRIPLET}",
-        "-DCMAKE_CXX_COMPILER=cl.exe",
-        "-DGOOGLE_CLOUD_CPP_ENABLE_WERROR=ON"
```

Comment:
Is this removal related?

Reply:
No, let me revert that and send it separately.  Turns out `..._WERROR=ON` when the quickstarts are built separately:

```
CMake Warning:
  Manually-specified variables were not used by the project:

    GOOGLE_CLOUD_CPP_ENABLE_WERROR
```

https://source.cloud.google.com/results/invocations/4f5bcf08-e111-4241-8742-be883c4a5e47/targets/cloud-devrel%2Fclient-libraries%2Fcpp%2Fgoogle-cloud-cpp%2Fmain%2Fwindows%2Fquickstart-cmake-static/log